modules/javafx.web/src/main/java/com/sun/webkit/network/PublicSuffixes.java

```diff
@@ -59,8 +66,7 @@
     /**
      * The mapping from domain names to public suffix list rules.
      */
-    private static final Map<String,Rule> RULES =
-            loadRules("effective_tld_names.dat");
+    private static final Map<String, Rules> rulesCache = new ConcurrentHashMap<>();
```

Comment:
I'm not sure that this needs to be concurrent, especially if you initialize it in a static init block (see my comment below).